tests/unit/cluster/failure-marking.tcl

```diff
@@ -0,0 +1,53 @@
+# Test a single primary can mark replica as `fail`
```

Comment:
This test isn't being run, need to add it here: https://github.com/redis/redis/blob/unstable/tests/test_helper.tcl#L100.